src/de.pengutronix.rauc.Installer.xml

```diff
@@ -1,3 +1,4 @@
+<!-- License note: This file is considered data and thus not covered by the project's main license. -->
```

Comment:
What do you think about:
```suggestion
<!-- License note: This file is considered data and thus no license is needed for any usage. -->
```

Reply:
I'd be fine with this, will update this PR in a few seconds...